README.md

```diff
@@ -205,6 +205,10 @@ To enable Google OAuth authentication, you need to set up Google OAuth in the Go
      - `GOOGLE_AUTH_CLIENT_ID`: Your Client ID
      - `GOOGLE_AUTH_CLIENT_SECRET`: Your Client Secret
 
+##### Google OAuth Troubleshooting
+
+If you encounter an error when signing in with google on the app (potentially after deleting data from your database) related to "oauth refresh token cannot be blank ... until you remove this website as a third party connection", you can fix this by going to the [Google Third-Party Apps and Services](https://myaccount.google.com/connections) and removing the app from the list of third-party apps with access to your account.
+
```

Comment:
I too have run into this issue. I had customized the error message a tiny bit to hopefully help, but I know it wasn't really that helpful. Rather than add this to the readme, I think I want to fix this once and for all. Especially since there may be cases when an actual production user runs into it.

There were two issues with the toast: (1) the toast error messages slide on and off the screen quickly so it's hard to put a lot in them, (2) you can't put a link in the toast error messages.

I've been meaning to refactor those toasts into real alert boxes that persist on the screen. I just went ahead and did that now. Check out this PR: https://github.com/AllYourBot/hostedgpt/pull/482

What do you think of this fix? Then we don't have to add instructions in the README.